client/src/components/Achievements/index.tsx

```diff
@@ -0,0 +1,74 @@
+import { useState, useEffect } from 'react';
+import { useStateContext } from "../../context";
+import { AcheiveItem } from './AcheiveItem';
+
+export const Achievements = () => {
+  const [campaignsData, setCampaignsData] = useState<any>(null);
+  const { address, contract } = useStateContext();
+
+  const donatedCampaigns = !campaignsData ? [] : campaignsData.filter((campaign: any) => campaign.donators.some((donator: string) => donator === address));
```

Comment:
1. Why `any`?
2. **donationCount** is a very strange name for array. Maybe **donationsList**?
3. A redundant negative operator. 

```suggestion
  const donatedCampaigns = campaignsData ? campaignsData.filter((campaign: any) => campaign.donators.some((donator: string) => donator === address)) : [];
```